dallinger/recruiters.py

```diff
@@ -77,6 +81,11 @@ class PsiTurkRecruiter(Recruiter):
 
     def __init__(self):
         """Set up the connection to MTurk and psiTurk web services."""
+
+        # commit to the db so if anything goes wrong the rest of the
+        # request is safe
+        session.commit()
```

Comment:
@thomasmorgan I see the reasoning behind wanting fewer lines and see that this produces the minimum, but it'll be easier to maintain with a style that's more explicit about which database changes need to be committed immediately before the recruiter is instantiated. It'll also help in case there are other unforeseen circumstances outside recruitment that also roll back the database.